apps/omg/lib/omg/utxo/position.ex

```diff
@@ -25,63 +25,67 @@ defmodule OMG.Utxo.Position do
   alias OMG.Utxo
   require Utxo
 
+  import Utxo, only: [is_position: 3]
+
   @type t() :: {
           :utxo_position,
           # blknum
-          pos_integer,
+          non_neg_integer,
```

Comment:
I relaxed this type due to dialyzer suggesting success typing for [get_position/1](https://github.com/omisego/elixir-omg/pull/756/files#diff-e24a6d0e13aa92e737647e776f74cadbR78) was able to return a `non_neg_integer` and wasn't entirely sure if `pos_integer` was necessary.